modules/nw-bgp-frr-k8s-migration.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * networking/bgp_routing/migrating-frr-k8s-resources.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-frr-k8s-migration_{context}"]
+= Migrating FRR-K8s resources
+
+You can migrate the FRR-K8s custom resources from the `metallb-system` namespace to the `openshift-frr-k8s` namespace.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+When upgrading from an earlier version of {product-title} with the Metal LB Operator deployed, you must manually migrate your custom frr-k8s configurations from the `metallb-system` namespace to the `openshift-frr-k8s` namespace. To move these CRs, enter the following commands:
```

Comment:
@asood-rh Could you help take a look this?

Reply:
@zhaozhanqi I tested these steps and bash script as well, no issues found.